2_process/src/munge_natural_baseflow.R

```diff
@@ -0,0 +1,81 @@
+munge_natural_baseflow <- function(baseflow_pred_files,segs_w_comids,vars=c("mean","med","p10","p90"),start_year,end_year){
```

Comment:
Could you add a logical argument `fill_all_years` that allows for using only monthly baseflow to fill all years? That would involve adding an if statement around `years_to_fill`

Reply:
Good idea, I'll add that argument!